bridges/darwinia-ethereum/bridge/src/service/affirm/handler.rs

```diff
@@ -159,17 +163,17 @@ impl AffirmHandler {
 
         tracing::trace!(
             target: "darwinia-ethereum",
-            "Prepare to affirm ethereum block: {}",
+            "[ethereum] [affirm] Prepare to affirm ethereum block: {}",
             target
         );
 
         match self.shadow.parcel(target as usize).await {
             Ok(parcel) => {
                 let parcel: EthereumRelayHeaderParcel = parcel.try_into()?;
-                if parcel.header == EthereumHeader::default() || parcel.mmr_root == [0u8; 32] {
+                if parcel.parent_mmr_root.to_fixed_bytes() == [0u8; 32] {
```

Comment:
To double check

Reply:
Before

https://github.com/darwinia-network/bridger/blob/c8c1a5babff6cc3d741fa4b26ce6d5b3e39c1ac6/frame/supports/support-ethereum/src/parcel.rs#L17


https://github.com/darwinia-network/bridger/blob/c8c1a5babff6cc3d741fa4b26ce6d5b3e39c1ac6/frame/supports/support-ethereum/src/block.rs#L15

After

https://github.com/darwinia-network/bridger/blob/2e4afda99f8bd9520f66b2714ccb1efb03e8a8f1/frame/assistants/darwinia-subxt/src/x_shadow.rs#L18